healthyregions/SDOHPlace
# UX Design Audit 11/14/2024

## Problem
We would like to make some last-minute UX improvements before user testing begins

## Approach
NOTE: This branch contains all changes from #377 and #378 as well

Met with Shubham a few times this week for design audit / live-coding sessions

Notes can be found here with full list of changes: https://www.notion.so/UX-Audit-2024-25-2a26f4fcfc394fe28150b5f26caacddc

## How to Test
1. Navigate to https://deploy-preview-380--cheerful-treacle-913a24.netlify.app/search
    * You should see the SDOH&Place Discovery App
    * You should see that the list of improvements above have been implemented
